tensorrt_llm/_torch/models/checkpoints/hf/qwen3vl_weight_mapper.py

```
@@ -6,3 +8,12 @@
 class Qwen3VLHfWeightMapper(HfWeightMapper):
     def preprocess_weights(self, weights: dict) -> dict:
         return weights
+
+    @property
+    def _head_dim(self):
+        config = self.model.config
+        if isinstance(config, Qwen3VLTextConfig):
+            return config.num_attention_heads
+        elif isinstance(config, Qwen3VLVisionConfig):
+            return config.num_heads
+        raise TypeError(f"Unexpected config class {type(config).__name__}.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)